### ISSUE TITLE
Crafting Recipe Loot Tables Not Randomizing Properly

### ISSUE THREAD

**Issue by TheVisionaryOne** [ghzdude/MCRandomizer#17]
After messing around with the crafting recipe randomizer for a bit, I encountered a bug where the loot tables only seem to randomize on the first world creation. After that, any new worlds created share the exact same randomized loot table as the first world.

The only current solution to this problem seems to be deleting the "randomizer" folder in config and re-launching the game.

**Comment by ghzdude** [comment]
the Recipe Randomizer uses a data structure that maps a vanilla item to different item. this data is then saved to `saves/world/data/` so that it only runs once. 

however, there is a null check when the server starts, functionally requiring a full restart to update the randomizations correctly.

**Comment by TheVisionaryOne** [comment]
Ah, that's unfortunate. It does seem like the randomizer works as you describe. I'm not entirely sure why it seemed necessary to delete the "randomizer" config folder, but it appears that is no longer necessary. I was able to successfully reset loot tables after simply restarting my game, just like you say. This is a bit tedious, but I suppose it's better than having to delete files before every world creation.

However, after extended testing, I encountered a separate issue, which I have submitted [here.](https://github.com/ghzdude/MCRandomizer/issues/19)

This issue makes world creation impossible after a game restart, conveniently enough...
